src/components/show-edit-pod/show-edit-pod.component.js

```diff
@@ -11,9 +10,12 @@ import StyledDeleteButton from "./delete-button.style";
 import Events from "../../utils/helpers/events";
 import { validProps } from "../../utils/ether";
 import tagComponent from "../../utils/helpers/tags";
+import LocaleContext from "../../__internal__/i18n-context";
 import StyledPod from "./show-edit-pod.style";
 
 class ShowEditPod extends React.Component {
+  static contextType = LocaleContext;
```

Comment:
Same question, why `contextType` in this component instead of `Consumer`.

Reply:
because I could use `contextType`. there is no `contextTypes`